src/test/java/com/webank/webase/front/gm/pressure/PerformanceHelloWorld.java

```diff
@@ -0,0 +1,211 @@
+package com.webank.webase.front.gm.pressure;
+
+import com.google.common.util.concurrent.RateLimiter;
+import com.webank.webase.front.gm.basic.HelloWorldGM;
+import org.fisco.bcos.channel.client.Service;
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.gm.GenCredential;
+import org.fisco.bcos.web3j.protocol.Web3j;
+import org.fisco.bcos.web3j.protocol.channel.ChannelEthereumService;
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.fisco.bcos.web3j.utils.Web3AsyncThreadPoolSize;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.math.BigInteger;
+import java.util.Random;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class PerformanceHelloWorld {
+  private static Logger logger = LoggerFactory.getLogger(PerformanceHelloWorld.class);
+  private static AtomicInteger sended = new AtomicInteger(0);
+
+  public static void main(String[] args) throws Exception {
+    try {
+      String groupId = args[3];
+      ApplicationContext context = new ClassPathXmlApplicationContext("classpath:applicationContext.xml");
+      Service service = context.getBean(Service.class);
+      service.setGroupId(Integer.parseInt(groupId));
+      service.run();
+
+      System.out.println("Start test...");
+      System.out.println("===================================================================");
+
+      ChannelEthereumService channelEthereumService = new ChannelEthereumService();
+      channelEthereumService.setChannelService(service);
+
+      Web3AsyncThreadPoolSize.web3AsyncCorePoolSize = 3000;
+      Web3AsyncThreadPoolSize.web3AsyncPoolSize = 2000;
+
+      ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(500);
+      Web3j web3 = Web3j.build(channelEthereumService, 15 * 100, scheduledExecutorService, Integer.parseInt(groupId));
+
+      Credentials credentials = Credentials.create("b83261efa42895c38c6c2364ca878f43e77f3cddbc922bf57d0d48070f79feb6");
+
+      BigInteger gasPrice = new BigInteger("30000000");
+      BigInteger gasLimit = new BigInteger("30000000");
+
+      String command = args[0];
+      Integer count = 0;
+      Integer qps = 0;
+
+      switch (command) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)